src/modules/user/order/components/OrderCard.tsx

```diff
@@ -79,4 +80,12 @@ const mapStatusToColor: Record<OrderStatus, TagProps["color"]> = {
   Cancel: "red",
 };
 
+const mapOrderStatusTranslation: Record<OrderStatus, string> = {
+  "In queue": "อยู่ในคิว",
+  Preparing: "กำลังเตรียมการ",
+  Ready: "รอรับออเดอร์",
```

Comment:
I don't know ""รอรับออเดอร์" is good word or not

Reply:
Okay, back to พร้อมเสิร์ฟ

Reply:
Maybe it's better with more accomplices @GolfGrab @katisd